css/src/assets/styles/cool.css

```diff
@@ -0,0 +1,192 @@
+html, body, div, p, footer, section{
+  padding: 0;
+  margin: 0;
+  border: none;
+  cursor:pointer;
+  font-family: 'Roboto Sans', sans-serif;
+  font-size: inherit;
+  color: #C3B7B5
+}
+
+.group:after {
+  content:" ";
+  display: block;
+  clear: both;
+
+}
+html {
+  background:  #C3B7B5;
+  min-width: 450px
+}
+
+body {
+
+  z-index: -2;
+  background: white;
+  width: 90%;
+  margin: 20px auto;
+  border-radius: 15px;
+}
+
+.frontpage {
+
+  display: block;
+  margin: 0 auto 10px auto;
+  padding: 20px 0;
+  height: 440px;
+  background: white;
+  position: relative;
+  border-radius: 15px;
+}
+
+
+#loose {
```

Comment:
I'd recommend avoiding applying of styles to `id`'s. Using `id`'s for styles can cause specificity headaches down the road when working on a large application.